src/util.ts

```diff
@@ -393,3 +393,27 @@ const addGraphTotals = (
   )
   return { totalTxs, totalUsd }
 }
+
+export const movingAveDataSort = (
+  data: Data[]
+): Array<{ date: string; allTxs: number }> => {
+  const aveArray: Array<{ date: string; allTxs: number }> = []
+  if (Array.isArray(data) === false) return aveArray
+  const dataLength = data.length
+  let sevenDaySum: number = 0
+  for (let i = 0; i < dataLength; i++) {
+    for (let j = i; j >= i - 6; j--) {
+      if (j < 0) {
+        sevenDaySum += 0
```

Comment:
Does not seem like `sevenDaySum` will change since we're only adding 0 (zero). Delete this line if it's not needed.

Reply:
Changed according to comment.